static/js/settings_config.js

```diff
@@ -320,3 +320,17 @@ exports.all_notifications = () => ({
         enable_online_push_notifications: !page_params.realm_push_notifications_enabled,
     },
 });
+
+exports.get_playground_for_languages = function (lang) {
+    // TODO: This is being hardcoded for now just for the prototype, post which we should
+    // add support for realm admins to configure their own choices.
```

Comment:
Just as as note on what we'll likely want as the eventual configuration API, I think we want each language to map to a list of objects, each object having a `name` and `url_pattern` fields, with `url_pattern` having a `{code}` parameter in it to substitute.